logger.py

```
@@ -0,0 +1,7 @@
+try:
+    risky_code()
+except:
+    pass  # Bad: no specific exception
```

[SEMANTIC_RULES | ERROR] Uninformative Exception Handling → Log the specific exception or take appropriate action instead of silently ignoring it.